crates/typst-cli/src/args.rs

```diff
@@ -155,6 +157,26 @@ pub struct SharedArgs {
     pub diagnostic_format: DiagnosticFormat,
 }
 
+/// An input path that is either stdin or a real path.
+#[derive(Clone, Debug)]
+pub enum InputPath {
+    /// stdin, represented by `-`
+    Stdin,
+    /// A non-empty path
+    Path(PathBuf),
+}
```

Comment:
```suggestion
/// An input path that is either stdin or a real path.
#[derive(Debug, Clone)]
pub enum InputPath {
    /// Stdin, represented by `-`.
    Stdin,
    /// A non-empty path.
    Path(PathBuf),
}
```
would the name `Input` perhaps be more fitting than `InputPath`?

Reply:
Makes sense, it's not always a path.